Code/Combat/WeatherMgr.cpp

```diff
@@ -1051,7 +1051,8 @@ void WeatherSystemClass::Render (RenderInfoClass &rinfo)
 
 		#if WEATHER_PARTICLE_SORT
 		#else
-		DX8Wrapper::Set_DX8_Render_State (D3DRS_ZBIAS, 12);
+		float depthbias = -0.02f;
+		DX8Wrapper::Set_DX8_Render_State(D3DRS_DEPTHBIAS,*reinterpret_cast<unsigned*>(&depthbias));
```

Comment:
This kind of type punning shenanigans will fall foul of strict aliasing in clang and GCC down the road, but since this is fairly strongly windows only code should we worry too much about it?

Reply:
This PR is doing the minimum to get the game running on D3D9.
My hope is to abstract the internals away such that these idiotic casts are not required at all.

On point, I don't think there's an aliasing issue here.
We don't have 2 variables pointing to the same value (e.g. a `float` and `char` pointer).